default/bash/aliases

```diff
@@ -8,6 +8,14 @@ alias cd="zd"
 zd() {
```

Comment:
(unrelated note)

The definition of `zd` and others here in the `aliases` file probably belongs with other function definitions in `default/bash/functions`.